components/esp_lvgl_port/include/esp_lvgl_port_disp.h

```diff
@@ -44,20 +44,27 @@ typedef struct {
     uint32_t    buffer_size;    /*!< Size of the buffer for the screen in pixels */
     bool        double_buffer;  /*!< True, if should be allocated two buffers */
     uint32_t    trans_size;     /*!< Allocated buffer will be in SRAM to move framebuf */
+    void        *user_buf1;     /*!< OPTIONAL: A buffer to be used by LVGL to draw the image */
+    void        *user_buf2;     /*!< OPTIONAL: Optionally specify a second buffer to render and flush */
```

Comment:
1. It seems, that we still add new items in this structure. Here is lot of buffers and I think, that it is not clear for custommers. We should add new union for specific settings for MIPI, RGB, monochromatic, ... 
2. Same for flags. 
3. I would like to add something into esp_lcd, how to determine, what bus is used (MIPI, RGB, ... )

Reply:
> 1. It seems, that we still add new items in this structure. Here is lot of buffers and I think, that it is not clear for custommers. We should add new union for specific settings for MIPI, RGB, monochromatic, ...
> 2. Same for flags.
> 3. I would like to add something into esp_lcd, how to determine, what bus is used (MIPI, RGB, ... )

Yes,  the fbs of SPI, I80 are allocated by esp_lv_port, not in the driver.
But the RGB driver will allocate fb internally, which depends on the user, and the internal buffer is more efficient.

The screen is initialized in BSP, at this time, the fb inside the RGB driver has been allocated. 
It is not appropriate to take it completely from inside the RGB, users may want to use thire own buffer, so it is a problem.